.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
   - "$HOME/.m2"
 script:
   - |
-    mvn clean package -q -B
+    mvn clean package -B
```

Comment:
Be careful with disabling `-q` because if the output becomes too long it will cause Travis to error out and fail the build.

Reply:
https://docs.travis-ci.com/user/common-build-problems/#log-length-exceeded

Reply:
I believe what I have done in the past is to run the phases seperately. For example:

```
script:
  - |
    mvn clean compile -B -q
    mvn test -B
    mvn package -B -q

Reply:
Travis run with the maven phases broken out seems good!
https://travis-ci.org/jacobsee/sonar-auth-openshift/builds/605764703